src/puppetlabs/trapperkeeper/core.clj

```diff
@@ -173,10 +173,12 @@
           (internal/parse-cli-args!)
           (run))
       (catch map? m
-        (case (without-ns (:type m))
-          :cli-error (quit 1 (:message m) *err*)
-          :cli-help (quit 0 (:message m) *out*)
-          (throw+)))
+        (let [type (:type m)]
+          (if (keyword? type)
```

Comment:
Oh wait, I think I was misunderstanding why the error was previously happening, which was the precondition of `keyword?` in `without-ns`. Alright, this makes more sense now. I'm good with it as-is. Thanks @camlow325!